### PR TITLE
Add SSL support to MQTTSession

### PR DESCRIPTION
**Pull request by adolfo**
@aciidb0mb3r This PR adds support for connecting to SSL hosts. :shipit: ?

### CODE REVIEW

**Comment by f2m2rd** [comment]
nice 👍 , can we please get this merged?

File: SwiftMQTT/SwiftMQTT/MQTTSession.swift [hunk 1]
```diff
@@ -32,8 +32,8 @@ public class MQTTSession: MQTTSessionStreamDelegate {
     private var messagesCompletionBlocks = [UInt16 : MQTTSessionCompletionBlock]()
     private var stream: MQTTSessionStream
     
-    public init(host: String, port: UInt16, clientID: String, cleanSession: Bool, keepAlive: UInt16) {
-        stream = MQTTSessionStream(host: host, port: port)
+    public init(host: String, port: UInt16, clientID: String, cleanSession: Bool, keepAlive: UInt16, useSSL: Bool) {
```
Comment (aciidb0mb3r) [inline SwiftMQTT/SwiftMQTT/MQTTSession.swift]: can you add default value for useSSL = false so existing clients don't break

**Comment by aciidb0mb3r** [comment]
sorry to be nitpicky here but can also you squash first 4 commits into one?
```
Add SSL support to MQTTSession
Update tests with useSSL
Update readme with SSL
Update example project
```

**Comment by aciidb0mb3r** [comment]
This is great, thanks so much!

File: SwiftMQTT/SwiftMQTT/MQTTSession.swift (hunk 1 above)
Comment (adolfo) [inline SwiftMQTT/SwiftMQTT/MQTTSession.swift]: Will do

**Comment by adolfo** [comment]
@aciidb0mb3r I set the default value for `useSSL` and squashed commits

**Comment by aciidb0mb3r** [comment]
Awesome, Thanks!

**Pull request merged**